src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -39,6 +39,7 @@
         .protoVersionBound(packageConfig.protoVersionBound(language))
         .commonProtosVersionBound(packageConfig.commonProtosVersionBound(language))
         .authVersionBound(packageConfig.authVersionBound(language))
+        .packageName(packageConfig.packageName(language))
```

Comment:
This is changed to:

    .packageName(packageConfig.packageName(language))
    .protoPackageName("proto-" + packageConfig.packageName(language))